command-line.md

```diff
@@ -6,7 +6,7 @@ In den meisten Filmen werden Hackers vor einem Bildschirm gezeigt der etwa so au
 
 Normalerweise ist es nichts anderes als eine Kommandozeile/ein Terminal/eine Konsole/ein Prompt. Das ist ein Tool, das es dem Benutzer erlaubt, den Computer mit textbasierten Kommandos zu steuern. Anstatt auf einen Ordner zu klicken, um zu schauen welche Dateien darin sind, gibt man ein Kommando in der Kommandozeile ein ('ls' in diesem Fall). Du kannst die Kommandozeile aber auch benutzen, um Textdateien zu bearbeiten, Dateien zu verschieben, zu löschen oder aus dem Internet runterzuladen, Programme zu installieren und viele andere Dinge. Im Prinzip kannst du alles, was du mit der Maus tust, auch mit der Kommandozeile machen (ausser zu Zeichnen wahrscheinlich). Entwickler (und Hacker) benutzen sie weil man oft viel schneller ist, wenn man nur Text eintippen muss, und nicht jedesmal zur Maus wechseln muss (natürlich erst wenn man die Kommandos einmal kennt).
 
-Und jetzt kannst du auch lernen, wir man damit arbeitet. Folge einfach dem Kapitel ["Einführung in die Kommandozeile"](https://tutorial.djangogirls.org/de/intro_to_command_line/)!
+Und jetzt kannst auch du lernen, wir man damit arbeitet. Folge einfach dem Kapitel ["Einführung in die Kommandozeile"](https://tutorial.djangogirls.org/de/intro_to_command_line/)!
```

Comment:
Gibt es einen Grund wieso den Tippfehler direkt nachher nicht auch noch korrigiert hast? "wie man damit arbeitet" anstatt "wir man damit arbeitet"?

Reply:
Wurde schon [in PR #10](https://github.com/django-girls-switzerland/static-website-tutorial-german/pull/10/files#diff-f5b3598e06375192c7baa31b591004b4R9] gefixed. Aber ich sollte den daraus folgenden Merge-Konflikt wohl noch beheben. Mach' ich gleich.